### PR TITLE
Add Ore Developer Role (Closes #484)

### LINKED ISSUE #484: Add ore developer role
An ore developer role should be added that has access to view all debugging/state information, but doesn't add the ability to perform moderation actions

### PR DESCRIPTION
**Pull request by phase**
Closes #484

I have no idea what needs to be done on the Forum side.

### CODE REVIEW

**Comment by felixoi** [comment]
#484 says "which has access to view all debugging/state information, but doesn't add the ability to perform moderation actions". This PR only provides default permissions to the ore devs. This doesn't fix the issue.

**Comment by phase** [comment]
oof, will do that later

File: app/ore/permission/role/RoleTypes.scala [hunk 1]
```diff
@@ -22,6 +22,7 @@ object RoleTypes extends Enumeration {
   val CommunityLeader = new  RoleType( 4, 59, classOf[GlobalRole], Default,  "Community Leader",   Amber)
   val Staff           = new  RoleType( 5, 3,  classOf[GlobalRole], Default,  "Sponge Staff",       Amber)
   val SpongeDev       = new  RoleType( 6, 41, classOf[GlobalRole], Default,  "Sponge Developer",   Green)
+  val OreDev          = new  RoleType(26, 65, classOf[GlobalRole], Default,  "Ore Developer",      Orange)
```
Comment (progwml6) [inline app/ore/permission/role/RoleTypes.scala]: this should be a group id of 66

**Review by progwml6 — CHANGES_REQUESTED** [review]
this also needs to provide the view access discussed to the developers

File: app/ore/permission/role/RoleTypes.scala (hunk 1 above)
Comment (phase) [reply app/ore/permission/role/RoleTypes.scala]: Is 65 being used somewhere?

Comment (Ewoutvans) [reply app/ore/permission/role/RoleTypes.scala]: group id 65 is Issue_Manager on the forums

Comment (phase) [reply app/ore/permission/role/RoleTypes.scala]: Remind me to document this in the code.

**Comment by phase** [comment]
I changed the role to `66`.

**Comment by phase** [comment]
It's a little hacky but it's the best way to implement it with the current permission system.

File: app/ore/permission/role/RoleTypes.scala [hunk 2]
```diff
@@ -22,6 +22,7 @@ object RoleTypes extends Enumeration {
   val CommunityLeader = new  RoleType( 4, 59, classOf[GlobalRole], Default,  "Community Leader",   Amber)
   val Staff           = new  RoleType( 5, 3,  classOf[GlobalRole], Default,  "Sponge Staff",       Amber)
   val SpongeDev       = new  RoleType( 6, 41, classOf[GlobalRole], Default,  "Sponge Developer",   Green)
+  val OreDev          = new  RoleType(26, 66, classOf[GlobalRole], Default,  "Ore Developer",      Orange)
```
Comment (felixoi) [inline app/ore/permission/role/RoleTypes.scala]: The ID for OrganisationAdmin is 26 too. This ends up in an error.

Comment (phase) [reply app/ore/permission/role/RoleTypes.scala]: :man_facepalming:

**Review by felixoi — APPROVED** [review]

**Pull request merged**